md/installation_guide.md

```diff
@@ -94,6 +94,16 @@ openssl pkcs12 -export -in /etc/letsencrypt/live/{domain}/fullchain.pem -inkey /
 
 Then when using the installer provide this generated file as a keystore, the same password for the keystore and the key, and the same alias.
 
+#### Renew certificate
+
+When the certificate expires, it can be renewed by reinstalling the backend (not the storage).
+
+* stop the backend using `bonita-ici stopApp`
+* start and deploy the backend using either the new certificate or newly generated one using `bonita-ici startApp deploy`
+
+If the certificate is autogenerated, the Bonita platform JVM needs to be restarted.
+
+
 ### Stop the module 
```

Comment:
We talk about the stopping the add-on for updating the certificate before we tell Patty how to stop the add-on.
Maybe updating should come after stopping?
Wording: "Stop the Add-on"

Reply:
yep maybe

Reply:
in fact its not the same title section +  I prefere  keeping "update the certificate" close to "security configuration"

Reply:
ok